test/dummy/db/schema.rb

```diff
@@ -124,12 +124,14 @@
   end
 
   create_table "spina_navigation_items", id: :serial, force: :cascade do |t|
-    t.integer "page_id", null: false
+    t.integer "page_id"
     t.integer "navigation_id", null: false
     t.integer "position", default: 0, null: false
     t.string "ancestry"
     t.datetime "created_at", precision: nil
     t.datetime "updated_at", precision: nil
+    t.string "url"
+    t.string "url_label"
```

Comment:
Why not `menu_title` like for pages? How do you handle translations?

Reply:
We ended up making changes so there is a `url_title` attribute that get translated through mobility.